module/trie.c

```diff
@@ -170,6 +183,30 @@ int trie_insert_string(struct trie *t, char *word)
     }
 }
 
+/*
+    Checks if a char exists in a trie 
+    Parameters:
+     - t: A pointer to the given trie
+     - c: The character we want to check
+
+    Returns:
+     - EXIT_SUCCESS if c exists in t
+     - EXIT_FAILURE if it doesn't
+*/
+int trie_char_exists(struct trie *t, char c) 
+{
+    assert(t != NULL);
+    assert(t->charlist != NULL);
+
+    int index = (int)c;
+
+    if (t->charlist[index] == '\0') {
```

Comment:
By having this function return a `bool`, we can condense these four lines into `return t->charlist[index] == '\0'`